pkg/asset/machines/openstack/machines.go

```diff
@@ -105,7 +96,57 @@ func Machines(clusterID string, config *types.InstallConfig, pool *types.Machine
 		machines = append(machines, machine)
 	}
 
-	return machines, nil
+	replicas := int32(total)
+
```

Comment:
we need to remove the FD fields before, see https://github.com/openshift/installer/blob/master/pkg/asset/machines/aws/machines.go#L125-L126